test-infra/camel-test-infra-aws-v2/src/test/resources/container.properties

```diff
@@ -0,0 +1,18 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+## tested against 2.1.2, 2.2.0 & 2.3.1
+aws.container=localstack/localstack:2.3.2
```

Comment:
This is wrong.

Reply:
You mean there should be localstack/localstack:3.0.0 ?

Reply:
Yes, otherwise the SQS tests will fail.
https://github.com/apache/camel/pull/12315 

I fixed with this PR and updated to 3.0.2

Reply:
Thank you!